### PR TITLE
Update `contributors/devel/README.md` to reference `testing.md`

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
Added a missing reference.

Resolves https://github.com/kubernetes/kubernetes/issues/35253

### CODE REVIEW

File: contributors/devel/README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Guide](http://kubernetes.io/docs/admin/).
 
 * **Development Guide** ([development.md](development.md)): Setting up your development environment.
 
+* **Testing** ([testing.md](testing.md)): Testing.
```
Comment (gyliu513) [inline contributors/devel/README.md]: How about s/Testing/How to run unit, integration and end to end tests.

**Comment by MikeSpreitzer** [comment]
@gyliu513 thanks for the suggestion.  I have added that, and one more detail.

**Pull request merged**